### DISCUSSION TITLE
Phoneme Customization

### DISCUSSION THREAD

**Discussion by thanhvodc** [idiap/coqui-ai-TTS#152] [Q&A] [answered]
Hi there! First of all, thank you for the fantastic work.

I'm currently looking to customize phonemes in XTTS-v2 without additional finetuning or training.

For example, I'd like to specify phonemes in a **file.txt** (like the example below). Could you advise on any changes needed in the **config.json** or specific parameters to ensure XTTS-v2 recognizes and uses the entries in **file.txt**? Alternatively, if there's a reference guide on this setup, could you please point me in the right direction?

Example file.txt:

```
Iacocca AY AH K OW K AH
5R08K133254 F AY V AA R OW EY T K W AH N TH R IY TH R IY T UW F AY V F AO R
SoCal S OW K AE L
```

Thanks in advance!

**Comment by eginhard** [comment, marked as answer]
XTTS does not use phonemes at all, so this is not possible.

**Reply by thanhvodc** [reply]
Thanks a lot.
